safe/definitions/hazard_classifications.py

```diff
@@ -183,11 +182,10 @@
             'affected': True,
             'description': tr('The medium hazard class.'),
             'string_defaults': ['medium'],
-            # Not used because EQ algs take care of this
-            # 'displacement_rate': 0.0,
+            'displacement_rate': 1.0,
             # Not used because EQ algs take care of this
             # 'fatality_rate': 0.0,
-            'numeric_default_min': 7,
+            'numeric_default_min': 6,
```

Comment:
@Charlotte-Morgan You changed these values yesterday. But I need to rollback, the minimum is not included. Please confirm this is still correct.